BookManager-iOS/Extension/UIKit/UIViewExtension.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  UIViewExtension.swift
+//  BookManager-iOS
+//
+//  Created by TOUYA KAWANO on 2020/05/29.
+//  Copyright © 2020 Toya Kawano. All rights reserved.
+//
+
+import UIKit
+
+public extension UIView {
+    
+    /// Viewが初期位置にいるかどうかを返します。
+    var isFirstPosition: Bool {
+        return frame.origin.y.isZero
+    }
+    
+    /// ViewをSafeAreaいっぱいに広げます。
+    /// - Parameter safeArea: safeArea
+    func fillSafeArea(safeArea: UILayoutGuide) {
+        translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            topAnchor.constraint(equalTo: safeArea.topAnchor),
+            bottomAnchor.constraint(equalTo: safeArea.bottomAnchor),
+            leftAnchor.constraint(equalTo: safeArea.leftAnchor),
+            rightAnchor.constraint(equalTo: safeArea.rightAnchor)
+        ])
+    }
+    
+    /// Viewのポジションが移動されていたらリセットします。
+    /// - Parameter duration: 実行時間
+    func resetPositonIfNeeded(duration: TimeInterval) {
+        if isFirstPosition { return }
+        UIView.animate(withDuration: duration) { [weak self] in
+            guard let weakSelf = self else { return }
```

Comment:
```suggestion
            guard let self = self else { return }
```
現状のままでもいいけど、swift 4.2 からこう書けるっぽい

Reply:
これ知りませんでした！:open_mouth:
ありがとうございます。修正しました:bow:
https://github.com/tkawano-cq/BookManager-iOS/pull/1/commits/72778d10cd04ee05560b99321929fcfa7ba0770d

Reply:
Suggested  change から修正せずに自分で直してしまったので上記のコミットからご確認お願いいたします。:bow: